hack/dind-cluster.sh

```diff
@@ -104,9 +104,15 @@ function check-selinux() {
   fi
 }
 
+DIND_IMAGE="openshift/dind"
 IMAGE_REGISTRY="${OPENSHIFT_TEST_IMAGE_REGISTRY:-}"
+if [[ -n "${IMAGE_REGISTRY}" ]]; then
+  DIND_IMAGE="${IMAGE_REGISTRY}/${DIND_IMAGE}"
+fi
 IMAGE_TAG="${OPENSHIFT_TEST_IMAGE_TAG:-}"
-DIND_IMAGE="${IMAGE_REGISTRY}openshift/dind${IMAGE_TAG}"
+if [[ -n "${IMAGE_TAG}" ]]; then
+  DIND_IMAGE="${DIND_IMAGE}:${IMAGE_TAG}"
+fi
```

Comment:
It's hard to argue against better docs, but the image caching configuration is removed by #8520 and most of the remaining options are already shared with vagrant (contrib/vagrant/provision-config.sh).  Is there something in particular that prompted this comment?